lib/helper/settings/get_settings.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:clnapp/constants/user_setting.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+
+Future<Setting> getSettingsInfo() async {
+  Setting setting = Setting();
+
+  final prefs = await SharedPreferences.getInstance();
+
+  if (prefs.getInt('connectionClient') == null) {
+    return setting;
+  }
+
+  setting.connectionType = clients[prefs.getInt('connectionClient')!];
+
+  setting.path = prefs.getString('selectedPath')!;
+
+  setting.host = prefs.getString('host')!;
+
+  setting.nickName = prefs.getString('nickName')!;
+
+  return setting;
+}
```

Comment:
we can do better there but for now, I accept this, can you open an issue, and maybe later with time we improve the Setting storage?